lib/fika/parser.ex

```diff
@@ -72,6 +72,14 @@ defmodule Fika.Parser do
     |> label("boolean")
     |> Helper.to_ast(:boolean)
 
+  atom =
+    ":"
+    |> string()
+    |> ignore()
```

Comment:
Let's treat the parser combinator as an exception and use `ignore(string(":"))` here even though the common style guides say otherwise.